src/components/SearchableSelect/SearchableSelect.jsx

```diff
@@ -367,12 +378,12 @@ const SearchableSelect = createClass({
 		return visibleOptionsCount > 0 ? (
 			options
 		) : (
-			<DropMenu.Option isDisabled>
-				<span className={cx('&-noresults')}>
-					No results match "{searchText}"
+				<DropMenu.Option isDisabled>
```

Comment:
We are using prettier which should handle all this formatting stuff. I'd suggest setting up your editor to run the code through prettier on save

Reply:
will get the prettier set up and update the code